go-controller/pkg/libovsdbops/db_object_types.go

```diff
@@ -88,3 +99,62 @@ var ACLNetpolDefault = newObjectIDsType(acl, NetpolDefaultOwnerType, []ExternalI
 	// egress or ingress
 	PolicyDirectionKey,
 })
+
+var ACLMulticastNamespace = newObjectIDsType(acl, MulticastNamespaceOwnerType, []ExternalIDKey{
+	// namespace
+	ObjectNameKey,
+	// egress or ingress
+	PolicyDirectionKey,
+})
+
+var ACLMulticastCluster = newObjectIDsType(acl, MulticastClusterOwnerType, []ExternalIDKey{
+	// cluster-scoped multicast acls
+	// there are 2 possible TypeKey values for cluster default multicast acl: DefaultDeny and AllowInterNode
+	TypeKey,
+	// egress or ingress
+	PolicyDirectionKey,
+})
+
+var ACLNetpolNode = newObjectIDsType(acl, NetpolNodeOwnerType, []ExternalIDKey{
+	// node name
+	ObjectNameKey,
+	// exact ip for management port, every node may have more than 1 management ip
+	IpKey,
+})
+
+// ACLNetworkPolicy define a unique index for every network policy ACL.
+// ingress/egress + NetworkPolicy[In/E]gressRule idx - defines given gressPolicy.
+// ACLs are created for every gp.portPolicies:
+// - for empty policy (no selectors and no ip blocks) - empty ACL (see allIPsMatch)
+// OR
+// - all selector-based peers ACL
+// - for every IPBlock +1 ACL
+// Therefore unique id for a given gressPolicy is portPolicy idx + IPBlock idx
+// (empty policy and all selector-based peers ACLs will have idx=-1)
+var ACLNetworkPolicy = newObjectIDsType(acl, NetworkPolicyOwnerType, []ExternalIDKey{
+	// policy namespace+name
+	ObjectNameKey,
+	// egress or ingress
+	PolicyDirectionKey,
+	// gress rule index
+	GressIdxKey,
+	PortPolicyIndexKey,
+	IpBlockIndexKey,
+})
+
+var ACLNetpolNamespace = newObjectIDsType(acl, NetpolNamespaceOwnerType, []ExternalIDKey{
+	// namespace
+	ObjectNameKey,
+	// in the same namespace there can be 2 default deny port groups, egress and ingress
+	PolicyDirectionKey,
+	// every port group has default deny and arp allow acl.
+	TypeKey,
+})
+
+var ACLEgressFirewall = newObjectIDsType(acl, EgressFirewallOwnerType, []ExternalIDKey{
+	// namespace
+	ObjectNameKey,
+	// there can only be 1 egress firewall object in every namespace, named "default"
+	// The only additional id we need is index of the ef rule
+	RuleIdx,
```

Comment:
Could you add a bit more insight of what this index represents in k8s terms in the comment? Is it just an index that is allocated based on when the rule was created and how many rules already existed at that time?

Reply:
egress firewall has a list of rules, and it is sorted `EgressFirewall.Spec.Egress []EgressFirewallRule`
so that id is the index of the rule represented with given acl.
I tried to update the comment, but not sure if it is more clear now. lmk if you have better ideas for this comment :)